### DISCUSSION TITLE
Match media keys with the respective tweet?

### DISCUSSION THREAD

**Discussion by fmafelipe** [tweepy/tweepy#1898] [Questions] [answered]
how can I match the information of the expansion "attachments.media_keys" with its respective tweet with obtained with "search_recent_tweets"

**Comment by Harmon758** [comment, marked as answer]
Iterating over a call to [`Paginator`](https://tweepy.readthedocs.io/en/v4.9.0/v2_pagination.html#tweepy.Paginator) should provide [`Response`](https://tweepy.readthedocs.io/en/v4.9.0/client.html#tweepy.Response) objects that have an `includes` (not `include`) field.

You can use the [`Tweet.attachments`](https://tweepy.readthedocs.io/en/v4.9.0/v2_models.html#tweepy.Tweet.attachments) attribute/field to retrieve the Tweet's `media_keys`.

**Reply by fmafelipe** [reply]
Thanks
